CenterForOpenScience/osf.io
# Adding a utility to prevent bad actors [ENG-2540] [WIP]

## Purpose

Bad actors are doing bad things on our site. This new service should help prevent those bad actors from doing bad things. Note this is still a WIP. I am waiting for the credentials to the service to fully test capability.

## Changes

Adding an adapter for the new service. Making the model utilize the new service

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify this prevents bad actors
- Verify

What are the areas of risk? N/A

Any concerns/considerations/questions that development raised? N/A

## Documentation

N/A

## Side Effects



## DevOps Notes

Theres some new settings:
`AKISMET_ENABLED` Should be set to true. This was created for testing purposes.
`OOPSPAM_APIKEY` must be filled in for OOPSpam to function

## Ticket

https://openscience.atlassian.net/browse/ENG-2540
